gcore/src/exec.rs

```diff
@@ -80,6 +80,29 @@ pub fn block_timestamp() -> u64 {
     timestamp
 }
 
+/// Get current cost multiplier.
+///
+/// The cost multiplier is a value that is used to calculate the cost of
+/// BLA-BLA-BLA.
```

Comment:
Some description is needed here. So far I can't express what this syscall is for

Reply:
Any updates on this research?